vision_language_model:
  ros__parameters:
    vlm_api_type: "ollama"
    vlm_api_model: "gemma3:4b"
    vlm_api_host: "jetson:11434"
    llm_temperature: 0.0